src/estimagic/optimization/tranquilo/fit_models.py

```diff
@@ -49,57 +46,30 @@ def get_fitter(fitter, user_options=None, model_info=None):
         "powell": fit_powell,
     }
 
-    if isinstance(fitter, str) and fitter in built_in_fitters:
-        _fitter = built_in_fitters[fitter]
-        _fitter_name = fitter
-    elif callable(fitter):
-        _fitter = fitter
-        _fitter_name = getattr(fitter, "__name__", "your fitter")
-    else:
-        raise ValueError(
-            f"Invalid fitter: {fitter}. Must be one of {list(built_in_fitters)} or a "
-            "callable."
-        )
-
     default_options = {
         "l2_penalty_linear": 0,
         "l2_penalty_square": 0.1,
+        "model_info": model_info,
```

Comment:
Yes, the naming is not great there. What I called `mandatory_arguments` can refer to `options` (static, partialled) or `arguments` (provided at runtime). Should we call it `mandatory_signature` which would be in line with the `signature` argument we have in some private functions? I avoided the word because we would not require a `inspect.Signature` object.